docs/tutorials/notebooks/Unbalanced_OT.ipynb

```diff
@@ -0,0 +1,381 @@
+{
```

Comment:
<ul><li>OTT -&gt; <code>OTT</code> using markdown</li><li>Use markdown for code variable names: <code>tau_a</code> and <code>tau_b</code> .</li><li>Better use directly these variables in the minimization problem ? (Maybe the formula ends up much more complicated)</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/291/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>For the <code>OTT</code> using markdown, I'd prefer to have {mod}<code>ott</code> as it links to the docs.  </p>

Reply:
<p>I did it with markdown because the other notebooks use markdown but I can change it if you prefer. Should I write {mod}<code>ott</code> instead ? </p>

Reply:
Changed it, thanks!